ui/app/AppLayouts/Wallet/popups/swap/SwapApproveCapModal.qml

```diff
@@ -44,6 +43,7 @@ SignTransactionModalBase {
     property string serviceProviderName: "Paraswap"
     property string serviceProviderURL: Constants.swap.paraswapUrl // TODO https://github.com/status-im/status-desktop/issues/15329
     property string serviceProviderContractAddress: "0x1bD435F3C054b6e901B7b108a0ab7617C808677b"
```

Comment:
not changed in this PR, but where did this contract address come from?
The one we interact with (approval, swap) is `0x216b4b4ba9f3e719726886d34a177484278bfcae` (fortunately the same address on every chain)
![image](https://github.com/user-attachments/assets/72cc8187-740c-432c-92da-5de955682af4)

Reply:
Ideally this should be coming from the backend, but in the meantime we probably want to put the write one here :D

Reply:
Agh, commented too early, it's overriden when instantiating the component. Nevermind!